### DISCUSSION TITLE
Open multiple mailto links

### DISCUSSION THREAD

**Discussion by FSchmidt-FUNKE** [zauberzeug/nicegui#3554] [Q&A]
### Description

Hi,

I'm a bit stuck and wanted to confirm whether this is possible before giving up on the idea. What I'm trying to achieve seems simple: I need to open a mailto link twice. My default mail client is Outlook, and after clicking the button, I'd like to see two separate _new mail_ windows open in Outlook.

Is there a way to accomplish this? Any guidance or suggestions would be greatly appreciated. Thanks!
```py
from nicegui import ui


textarea = ui.textarea(label='Body').classes('w-1/2')
def __mailto():
    #.replace(" ", "%20").replace("\n", "%0A")
    prepared_body = textarea.value.replace('\n', '%0D%0A')

    mailto_link = ('mailto:blabala@dummy.com'
                   '?subject=BlubBlub'
                   f'&body={prepared_body}'
                   )

    ui.navigate.to(mailto_link, new_tab=False)
    ui.navigate.to(mailto_link, new_tab=False)

ui.button('Mailto', on_click=__mailto)

ui.run()
```

**Comment by rodja** [comment]
I think this might not have to do anything with NiceGUI but more with HTML and the specifics of the browser.

**Comment by FSchmidt-FUNKE** [comment]
True, there is a difference in behaivor between browsers
Especially in Firefox the behavior is quite weird. If I replace the second `ui.navigate.to()` with an `ui.notify('hey')` there is also no notification.
Setting a different textarea value after `ui.navigate.to()` is working tho. For Chrome or Edge the notification get's shown.

Using Firefox I also managed to open multiple new message windows in Outlook:
```python
from nicegui import ui


textarea = ui.textarea(label='Body').classes('w-1/2')
def __mailto():
    prepared_body = textarea.value.replace('\n', '%0D%0A')

    mailto_link = ('mailto:blabala@dummy.com'
                   '?subject=blub'
                   f'&body={prepared_body}'
                   )

    ui.navigate.to(mailto_link, new_tab=False)
    ui.timer(interval=2, callback=lambda: ui.navigate.to(mailto_link, new_tab=False), once=True)
    ui.timer(interval=4, callback=lambda: ui.navigate.to(mailto_link, new_tab=False), once=True)


ui.button('Mailto', on_click=__mailto)

ui.run()
```
Interestingly, it seems that there has to be at least a delay of two seconds. Unfortunatley this solution still isn't working for Edge/Chrome. Both of them still only open one window in Outlook. When using `new_tab=True` it works even without a timer in Firefox, but still only in Firefox
